store/tikv/oracle/oracle.go

```diff
@@ -141,3 +141,12 @@ func GoTimeToTS(t time.Time) uint64 {
 	ts := (t.UnixNano() / int64(time.Millisecond)) << physicalShiftBits
 	return uint64(ts)
 }
+
+// MaxTxnTimeUse is the max time a Txn may use (in ms) from its begin to commit.
+// We use it to abort the transaction to guarantee GC worker will not influence it.
+const MaxTxnTimeUse = 24 * 60 * 60 * 1000
```

Comment:
Seems not very suitable to move the const to here. Maybe we can pass it to `GoTimeToLowerLimitStartTS` as another parameter.

Reply:
Agree.